src/components/Greeting.js

```diff
@@ -0,0 +1,24 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { getRandomGreeting } from '../redux/greetings';
+
+const Greeting = () => {
+  const greeting = useSelector((state) => state.message);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(getRandomGreeting());
+  }, [dispatch]);
+
+  return (
+    <section>
+      <h1>Random Greeting</h1>
+      <h3>{greeting}</h3>
+      <Link to="/">
+        <button type="button">Back to Home</button>
+      </Link>
+    </section>
+  );
+};
+
+export default Greeting;
```

Comment:
- [ ] As required for the project your two apps to work together, kindly ensure that your frontend app displays a random greeting to the UI. Right now, it shows a blank page and shows an error in your console. Below, is the image of how it looks like:
![Screenshot from 2023-07-11 15-40-48](https://github.com/raihan2bd/hello-world-front-end/assets/83042742/3dad94cb-779c-4ec0-9952-9c874fde5bd3)

Reply:
Hi @tresorsawasawa 👋,  Thanks for your information, However, everything is working on my side
![Capture](https://github.com/raihan2bd/hello-world-front-end/assets/35267447/d6efadcc-ea5d-4b63-9b2c-7ffe85bedfb6)
Make sure your front-end port is `3456`
![Capture2](https://github.com/raihan2bd/hello-world-front-end/assets/35267447/28598bc4-ad2f-4029-9221-168be8544616)

Reply:
Hi @raihan2bd 

It it is now working, that is good then.